pkg/server/plugin/nodeattestor/x509pop/x509pop.go

```diff
@@ -121,6 +122,7 @@ func (p *X509PoPPlugin) Attest(stream nodeattestor.Attest_PluginStream) error {
 	resp := &nodeattestor.AttestResponse{
 		Valid:        true,
 		BaseSPIFFEID: x509pop.SpiffeID(c.trustDomain, leaf),
+		Selectors:    buildSelectors(leaf, chains),
```

Comment:
would you mind updating the unit-tests to assert the correct selectors are being returned?

Reply:
Sure! Sorry about that